oereb_einzelschutz_denkmal/update_oerebkrm_v1_1_localiseduri_table.sql

```diff
@@ -0,0 +1,14 @@
+UPDATE stage.oerebkrm_v1_1_localiseduri
+    SET atext = replace(atext, 'https://ch.so.ada.denkmal', 'https://ch.so.ada.denkmal-stage')
+WHERE
```

Comment:
Ich bin der Meinung, dass es die ganze WHERE-Bedingung nicht braucht. Ist es nicht so, dass die `replace`-Funktion die Ersetzung macht, falls es das Gesuchte findet, und nichts macht, falls es das Gesuchte nicht findet?

Ah, das "Problem" ist, dass dann alle anderen Zeilen, wo es nicht gefunden wird, unnötigerweise mit sich selbst geupdatet werden. Dann mach es doch einfach so:
```suggestion
WHERE
    atext LIKE '%ch.so.ada.denkmal%'
```
Dann updatet es ebenfalls nur diejenigen Objekte, wo überhaupt `ch.so.ada.denkmal` drin vorkommt.

Reply:
Ich habe das so gemacht, weil es ja theoretisch möglich wäre, dass der String 'https://ch.so.ada.denkmal' auch in einem anderen datatset im Attribut "atext" enthalten sein könnte.